src/language-html/visitor-keys.js

```diff
@@ -11,6 +11,8 @@ const visitorKeys = {
   comment: [],
   attribute: [],
   cdata: [],
+  block: ["children", "parameters", "successorBlock"],
```

Comment:
`successorBlock` is unnecessary?

Reply:
Indeed, I deleted it

Reply:
@fisker Can you remove `successorBlock` ?